package.json

```diff
@@ -1002,5 +1002,11 @@
             }
         ]
     },
-    "packageManager": "pnpm@8.6.0"
+    "packageManager": "pnpm@8.6.0",
```

Comment:
Make this the last property (aka move capabilities up)

Reply:
https://github.com/leonardssh/vscord/pull/261/commits/a4fae4311a3e4b1c9285a513a66434c2d4a0a64f